bot/client.js

```diff
@@ -10,13 +11,39 @@ const client = new Discord.Client();
 // Pre-load controllers
 const controllers = require('./controllers')();
 
+// Award bonus points
+const awardBonusPoints = async (user) => {
+  const numberOfMessagesForBonus = 1000;
+  const amountOfBonusPoints = 100;
+  // Get User Message Count
+  const memberData = await models.Member.findAll(
+    {
+      attributes: ['messagesCount', 'points'],
+      where: { discordUser: user.author.id },
+    },
+  );
+  let { messagesCount, points } = memberData[0].dataValues;
+  util.log('Results from database call', memberData[0].dataValues, 4);
+  // Check if its greater or equal to numberOfMessagesForBonus
+  if (messagesCount >= numberOfMessagesForBonus) {
+    points += amountOfBonusPoints;
+    messagesCount = 0;
```

Comment:
Quick input, I noticed this is going to affect the points that users gain from regularly chatting. That feature directly uses the messagesCount to determine how many points the users have earned through chatting. If there would be a way to implement this without resetting the message count to zero each time that would be great.